package.json

```diff
@@ -6,22 +6,25 @@
   "repository": "https://github.com/prettier/prettier-ruby",
   "author": "Amit Solanki <@iamsolankiamit>",
   "license": "MIT",
-  "files": ["src", "vendor"],
+  "files": [
+    "src",
+    "vendor"
+  ],
   "engines": {
     "node": ">=6"
   },
   "dependencies": {
     "prettier": "prettier/prettier#d05a29da05b2084790491eaef85917ce584e4fbd"
   },
   "devDependencies": {
-    "all-contributors-cli": "^4.11.2",
-    "eslint": "^4.14.0",
+    "all-contributors-cli": "^5.0.0",
```

Comment:
Disclaimer: I haven't tested this version, I can keep it pinned to `^4.11.2` if need be :+1: